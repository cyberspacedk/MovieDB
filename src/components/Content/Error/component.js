import React from 'react';
import { Layout, Alert } from 'antd';

const ErrorRequest = () => (
  <Layout>
    <Layout.Content>
      <div className="top-margin">
        <Alert
          message="OMG"
          description="Hey dude smth went wrong. Try again later"
          type="error"
          showIcon
        />
      </div>
    </Layout.Content>
  </Layout>
);

export default ErrorRequest;
