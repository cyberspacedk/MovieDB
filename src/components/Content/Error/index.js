import ErrorRequest from './component';

export default ErrorRequest;
