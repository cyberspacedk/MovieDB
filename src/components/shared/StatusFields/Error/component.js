/* eslint-disable no-shadow */
/* eslint-disable react/prop-types */
import React, { useEffect } from 'react';
import { Layout, Alert, Row, Col } from 'antd';
import { connect } from 'react-redux';
import { tryAgain } from '../../../../store/authentifiction/actions';

const ErrorRequest = ({ tryAgain }) => {
  useEffect(() => {
    setTimeout(() => {
      tryAgain();
    }, 2000);
  });

  return (
    <Layout>
      <Layout.Content>
        <Row type="flex" justify="center">
          <Col span={10}>
            <Alert
              message="OMG"
              description="Hey dude smth went wrong. Try again later"
              type="error"
              showIcon
            />
          </Col>
        </Row>
      </Layout.Content>
    </Layout>
  );
};
const mdtp = {
  tryAgain,
};
export default connect(
  null,
  mdtp,
)(ErrorRequest);
