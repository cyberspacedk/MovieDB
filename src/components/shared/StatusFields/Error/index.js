import Error from './component';

export default Error;
