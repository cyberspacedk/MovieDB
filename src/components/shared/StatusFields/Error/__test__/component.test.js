import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Error from '../component';

describe('Component: Error', () => {
  it('Match its snapshot.', () => {
    const wrapper = shallow(<Error />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
