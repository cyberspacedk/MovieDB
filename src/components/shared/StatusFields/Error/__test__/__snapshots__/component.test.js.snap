// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Error Match its snapshot. 1`] = `
<Adapter>
  <Adapter>
    <Row
      gutter={0}
      justify="center"
      type="flex"
    >
      <Col
        span={10}
      >
        <Alert
          description="Hey dude smth went wrong. Try again later"
          message="OMG"
          showIcon={true}
          type="error"
        />
      </Col>
    </Row>
  </Adapter>
</Adapter>
`;
